template/TEMPLATE_README.md

```diff
@@ -0,0 +1,67 @@
+# Package template for BrewBlox Service plugins
+
+There is some boilerplate code involved when creating a new Python package.
+
+The general idea is that you copy everything in here to your new repository or directory, update some placeholder names, and start coding.
```

Comment:
git clone/fork instead of copy?
Perhaps we should make this template a separate repository, so we can update it and plugin maintainers can pull the changes.

It will be easier to to fork this repo to have a base for a custom plugin.